README.rst

```diff
@@ -36,6 +36,13 @@ You may want to perform something like:
     $ source venv/bin/activate
     $ pip install -r requirements.txt
 
+It is also recommended to make linters run to keep up with coding standards
```

Comment:
@pfmoore any suggestion on wording improvements?

Reply:
No, not really. As I said, I'm not convinced we need linter checks on this project, so I'm hardly the one to say how you should word a recommendation that we use them :smile: